src/Features/CSharp/Portable/EditAndContinue/AbstractSyntaxComparer.cs

```diff
@@ -6,34 +6,22 @@
 using System.Collections.Immutable;
 using System.Diagnostics;
 using Microsoft.CodeAnalysis.Differencing;
-using Microsoft.CodeAnalysis.Text;
 namespace Microsoft.CodeAnalysis.CSharp.EditAndContinue
 {
-    internal abstract class AbstractSyntaxComparer : TreeComparer<SyntaxNode>
+    internal abstract class AbstractSyntaxComparer : Microsoft.CodeAnalysis.Differencing.AbstractSyntaxComparer
```

Comment:
>AbstractSyntaxComparer [](start = 28, length = 22)

Do we still have multiple subclasses of the C# comparer? If not we should merge the SyntaxComparer with this AbstractSyntaxComparer

Reply:
No, I meant to mention this at the top but forgot. Your suggestion is one of the potential solutions though, and the one I was leaning towards, so I'll go with it :)